asdf.fish

```diff
@@ -1,10 +1,13 @@
 set -x ASDF_DIR (dirname (status -f))
-set -l asdf_data_dir (
-  if test -n "$ASDF_DATA_DIR"; echo $ASDF_DATA_DIR;
-  else; echo $HOME/.asdf; end)
+
+if test -n $ASDF_DATA_DIR
+  set -l asdf_user_shims $ASDF_DATA_DIR/shims
+else
+  set -l asdf_user_shims $HOME/.asdf
```

Comment:
Shouldn't this be `$HOME/.asdf/shims`? The `$HOME/.asdf/` directory won't contain anything worth adding to the `PATH`.

Reply:
oh, I completely missed that, I'll fix it